### ISSUE TITLE
"Twitter" and "Github" logo should be little bright on the footer

### ISSUE THREAD

**Issue by kmlhsn** [HTTPArchive/almanac.httparchive.org#1636]
**URL**
1. Go to almanac.httparchive.org
2. Scroll down to bottom

**Describe the issue**
"Twitter" and "GitHub" logo should be liitle bright on the footer

**Expected behavior**

"Twitter" and "GitHub" logo should be little bright on the footer. logo color and footer background color almost same.
**Screenshots**
![image](https://user-images.githubusercontent.com/32615215/101260389-b7312380-3759-11eb-8f8c-a849ae8efc6a.png)

**Comment by rviscomi** [comment]
@bazzadp can you triage? I think this may be intentional and within our accessibility guidelines for contrast, but if not feel free to reassign to the development team.

**Comment by bazzadp** [comment]
Think it's not contrasted enough but we've been getting away with it from our automated testing because there's not text.

Son't see any reason why they should be a different colour than the HA Logo to be honest.
